src/vs/workbench/contrib/notebook/test/browser/notebookDiffViewModel.test.ts

```diff
@@ -1,404 +0,0 @@
-/*---------------------------------------------------------------------------------------------
- *  Copyright (c) Microsoft Corporation. All rights reserved.
- *  Licensed under the MIT License. See License.txt in the project root for license information.
- *--------------------------------------------------------------------------------------------*/
-
-import assert from 'assert';
-import * as sinon from 'sinon';
-import { CellKind } from 'vs/workbench/contrib/notebook/common/notebookCommon';
```

Comment:
All of these tests are pretty much duplicates of tests in `notebookDiff.test.ts.`
Updated `notebookDiff.test.ts` to include the missing tests